AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -168,6 +176,17 @@ - (void)signIn {
   }
 }
 
+- (void)signOut {
+  if ([MSAuthTokenContext sharedInstance].authToken != nil) {
+    MSLogInfo([MSIdentity logTag], @"User sign-out succeeded.");
+    [MSAuthTokenContext sharedInstance].authToken = nil;
```

Comment:
Topic to discuss:
It will raise the existing event with `nil` token. We can just re-use it.

Reply:
One more thing: it conflicts with #1328 😭